drdown/medicalrecords/models/model_curves.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from ...users.models.model_patient import Patient
+
+
+class Curves(models.Model):
+
+    patient = models.ForeignKey(
+        Patient,
+        on_delete=models.CASCADE,
+        verbose_name=_('Patient')
+    )
+    weight = models.FloatField(
+        default=0
+    )
+    height = models.IntegerField(
+        default=0
+    )
+    cephalic_perimeter = models.FloatField(
+        default=0
+    )
+    bmi = models.FloatField(
```

Comment:
O que é BMI?

Reply:
é o IMC?

Reply:
Se for o microserviço já faz o calculo, só passar via post o height e o weight

Reply:
BMI é o IMC, nós calculamos ele na aplicação também para poder exibir junto da tabela de dados. Eu achei legal fazer assim e que adiciona valor ao produto poder conferir o valor numérico além do gráfico. Não sei se a @marimendes concorda.

Reply:
Eu achei que agrega um valor legal.

Reply:
de boa :)

Reply:
O BMI precisa também da correção do float